src/hotspot/share/runtime/os.cpp

```diff
@@ -399,10 +399,8 @@ static void signal_thread_entry(JavaThread* thread, TRAPS) {
         // Any SIGBREAK operations added here should make sure to flush
         // the output stream (e.g. tty->flush()) after output.  See 4803766.
         // Each module also prints an extra carriage return after its output.
-        VM_PrintThreads op;
+        VM_PrintThreads op(tty, PrintConcurrentLocks, false, true);
```

Comment:
Please comment what the false/true arguments mean e.g.

... false /* no extended info \*/, true /* print JNI handle info \*/);

Reply:
fixed.